uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/BaseKeyCloakManager.java

```diff
@@ -56,6 +61,10 @@
 
     static final int STATUS_NOT_AUTHORIZED = 403;
 
+    protected static final String DEFAULT_USE_RESOURCE_ROLE_MAPPING = "false";
+    protected static final String CLIENT_ID = "kie";
```

Comment:
Is this a default value? Does client always have to be `kie`?

Reply:
@tomasdavidorg I guess It should be "kie" always. because this BC uses this identifier to find the client where business central URI is configured in RHSSO. We mention the same way in doc 
https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html-single/integrating_red_hat_process_automation_manager_with_red_hat_single_sign-on/index#sso-central-client-create-proc

Reply:
though i think this should be mentioned in our docs as a mandatory.